mongo/cursor.go

```diff
@@ -85,8 +85,31 @@ func newEmptyCursor() *Cursor {
 func (c *Cursor) ID() int64 { return c.bc.ID() }
 
 // Next gets the next result from this cursor. Returns true if there were no errors and the next
-// result is available for decoding.
+// result is available for decoding. Next blocks until an event is available for decoding or ctx expires.
```

Comment:
It seems error-prone (i.e. not DRY) to repeat these doc comments on Cursor and ChangeStream.  Is this normal in Go?  Should we define an interface that they both implement and move the comments to the interface?

Reply:
I didn't like the comment-repeat either. We originally had an interface (`Cursor`) but removed it in favor of a concrete type (`ChangeStream`). I don't think defining an interface that we don't actually return will be useful because a user clicking through docs will make their way to `ChangeStream`, not the interface.

Reply:
OK.  I guess this is the only place where we have this problem so it's not too bad.